deployments/rollup-bsn-demo/docs/BSN_DEMO_SETUP.md

```diff
@@ -0,0 +1,246 @@
+# Babylon BSN Integration Demo - Manual Setup Guide
+
+This guide walks through the complete manual setup of the Babylon BSN Integration Demo, including local blockchain infrastructure and finality providers for both Babylon and Anvil networks.
+
+## Important Version Note
+
+**Current main finality-provider commit `ff6b427d4c88bedcf826ee7cdcae06b2a49e4248` has an issue where the OP client relies on FP gadget.** 
```

Comment:
Should we create a github issue and have its link here?

Reply:
added, great catch a48234faa99146ff46adda540acbf28c7e40ac5e